packages/core/admin/admin/src/content-manager/pages/EditView/components/InputRenderer.tsx

```diff
@@ -145,4 +151,64 @@ const attributeHasCustomFieldProperty = (
 ): attribute is Attribute.Any & Attribute.CustomField<string> =>
   'customField' in attribute && typeof attribute.customField === 'string';
 
+const useFieldHint = (hint: ReactNode = undefined, attribute: Attribute.Any) => {
+  const { formatMessage } = useIntl();
+
+  const { maximum, minimum } = getMinMax(attribute);
+
+  if (!maximum && !minimum) {
+    return hint;
+  }
+
+  const units = !['biginteger', 'integer', 'number'].includes(attribute.type)
+    ? formatMessage(
+        {
+          id: 'content-manager.form.Input.hint.character.unit',
+          defaultMessage: '{maxValue, plural, one { character} other { characters}}',
+        },
+        {
+          maxValue: Math.max(minimum || 0, maximum || 0),
+        }
+      )
+    : null;
+
+  const minIsNumber = typeof minimum === 'number';
+  const maxIsNumber = typeof maximum === 'number';
+  const hasMinAndMax = maxIsNumber && minIsNumber;
```

Comment:
shouldn't min and max be a number or undefined at this point?
```suggestion
  const hasMinAndMax = maxIsNumber && minIsNumber;
```